app/templates/404.html

```diff
@@ -1,18 +1,22 @@
 {% if display_region == "ni" %}
     {% extends "base-ni.html" %}
+	{% set contact_us_link = '#' %}
```

Comment:
The indentation on this looks off. Mixed tabs and spaces?

Reply:
Correction, mixed tabs and spaces throughout this file.

Reply:
Reset for spaces